### PR TITLE
Updating Comments on Pages

### PR DESCRIPTION
**Pull request by lazpz61BU**
Updated the commenting on the pages files to give a description of what each function and render return is doing

### CODE REVIEW

**Review by saahilv — APPROVED** [review]
Comments look good

File: code/Frontend/react-frontend/src/pages/payment_validation.js [hunk 1]
```diff
@@ -1,6 +1,9 @@
 import React, { useState } from "react";
 import { Container, Form, Button, Alert, Col, Row } from "react-bootstrap";
 
+import {useLocation, useNavigate } from "react-router-dom";
```
Comment (smathis122) [inline code/Frontend/react-frontend/src/pages/payment_validation.js:4]: Can you remove this line -- it is a duplicate of line 7 and causes errors

**Review by smathis122 — COMMENTED** [review]
Looks good -- except for the one line in payment_validation.js

**Review by lazpz61BU — COMMENTED** [review]
The changes have been made

**Review by smathis122 — APPROVED** [review]

**Pull request merged**